validation/validator_test.go

```diff
@@ -174,11 +174,11 @@ func TestValidateFail(t *testing.T) {
 	proof, err := init.Initialize()
 	r.NoError(err)
 
-	proof.Identity = append([]byte{0}, proof.Identity[1:]...)
+	Identity := append([]byte{0}, id[1:]...)
```

Comment:
if you called this variable `wrongId` or `modifiedId` I think this test would become much clearer.